project/Build.scala

```diff
@@ -27,8 +27,8 @@ object ScaldingBuild extends Build {
   val chillVersion = "0.6.0"
   val dfsDatastoresVersion = "1.3.4"
   val elephantbirdVersion = "4.8"
-  val hadoopLzoVersion = "0.4.16"
-  val hadoopVersion = "1.2.1"
+  val hadoopLzoVersion = "0.4.19"
+  val hadoopVersion = "2.5.0"
```

Comment:
I rolled in another PR people had green since it effected testing time hugely, so two birds one stone. People had reviewed independently and thought ok.